bundles/org.openhab.binding.netatmo/src/main/resources/OH-INF/i18n/netatmo.properties

```diff
@@ -305,6 +311,13 @@ channel-type.config.netatmo.live-stream-url.quality.option.low = Low Quality
 channel-type.config.netatmo.live-stream-url.quality.option.poor = Poor Quality
 channel-type.config.netatmo.live-stream-url.quality.option.high = High Quality
 
+# channel group types
+
+channel-group-type.netatmo.siren.channel.status.label = Siren Status
+channel-group-type.netatmo.siren.channel.status.description = Status of the siren
+channel-group-type.netatmo.siren.channel.monitoring.label = Monitoring
+channel-group-type.netatmo.siren.channel.monitoring.description = Monitoring state of the equipment
+
```

Comment:
To be removed, you already have the entries added for the channel types.
When you have a separate block of entries like there, you can ask yourself if this is not entries previously added but which are no more needed.

Reply:
As a reminder, the i18n tool removes nothing, it just adds/updates entries.